File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/app/IQChannelsConfig.java
package ru.iqchannels.sdk.app;

import ru.iqchannels.sdk.Log;

public class IQChannelsConfig {
    public String address;
    public String channel;

    public IQChannelsConfig() {}

    public IQChannelsConfig(String address, String channel) {
        this(address, channel, true);
    }

    public IQChannelsConfig(String address, String channel, boolean logging) {
        this.address = address;
        this.channel = channel;
        Log.configure(logging);
    }
}

File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/app/IQChannelsConfig.kt
package ru.iqchannels.sdk.app

import ru.iqchannels.sdk.IQLog

class IQChannelsConfig @JvmOverloads constructor(
	val address: String?,
	val channels: List<String>?,
	val chatToOpen: String?,
	logging: Boolean = true,
	val uiOptions: UIOptions = UIOptions(),
) {

	init {
		IQLog.configure(logging)
	}
}
